benchmark/MainPrintTimes.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Andre Schlegel (November of 2022, schlegea@informatik.uni-freiburg.de)
+
+#include <iostream>
+#include <sstream>
+#include <algorithm>
+#include <vector>
+
+#include "../benchmark/Benchmark.h"
+#include "../benchmark/util/MainFunctionHelperFunction.h"
+
+/*
+ * @brief Goes through the registerd single measurment benchmarks, measures their time and
+ * prints their measured time.
+ */
+int main() {
+  // The descriptors and measured times of all the single measurment benchmarks.
+  const std::vector<BenchmarkRecords::RecordEntry> records = measureTimeForAllSingleMeasurments(); 
+ 
+  // Visualizes the measured times.
+  std::stringstream visualization;
+  visualization << "##############\n# Benchmarks #\n##############\n";
+  for (const BenchmarkRecords::RecordEntry& entry: records) {
+    visualization << "\nBenchmark '" << entry.descriptor << "' took " << entry.measuredTime << " seconds.";
+  }
+
+  std::cout << visualization.str() << "\n";
+};
```

Comment:
Change, so that all the new formats also get printed.

Reply:
First working draft in [75073cc](https://github.com/ad-freiburg/qlever/pull/860/commits/75073cccf758eb3bdb8f68e6ea8e6aa8b23def05).

Still needs to be checked for bugs and the code needs to be cleaned up.

Reply:
Got rid of some code duplication in:
- [e1b4ecc](https://github.com/ad-freiburg/qlever/pull/860/commits/e1b4ecceb3d2b349276d550a897a9714c550df65)
- [de5c95a](https://github.com/ad-freiburg/qlever/pull/860/commits/de5c95a99069f9e989ea40b59b2e43762281380d)

Reply:
Improved the way tables were drawn, so know a column should keep it's width. See [c2718b5](https://github.com/ad-freiburg/qlever/pull/860/commits/c2718b5091490d70a4648fd80f673629ed9f4adb).